jobs/config.json

```diff
@@ -8117,6 +8133,30 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-kubemark-100-canary": {
+    "args": [
+      "--cluster=kubemark-100",
+      "--docker-in-docker",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/ci-kubernetes-kubemark-100-gce.env",
+      "--env-file=jobs/ci-kubernetes-kubemark-100-canary.env",
+      "--extract=ci/latest",
+      "--gcp-zone=us-central1-f",
```

Comment:
It will pick up a free project from boskos, if it does not require special quota.